rotkehlchen/constants/assets.py

```diff
@@ -280,5 +280,7 @@
 
 A_WETH_OPT = Asset('eip155:10/erc20:0x4200000000000000000000000000000000000006')
 A_WETH_ARB = Asset('eip155:42161/erc20:0x82af49447d8a07e3bd95bd0d56f35241523fbab1')
+A_WETH_BASE = Asset('eip155:8453/erc20:0x4200000000000000000000000000000000000006')
```

Comment:
are they in the global DB also?

Reply:
A_WETH_BASE is not present, will it raise any error? I think it should not because it's not resolved anywhere. But we should add it, right?

Reply:
So did you add it? I don't see any globalDB diff here

Reply:
So, I see that it is added in the db on develop branch. Earlier I checked on production, where it was not added. Did you or Yabir add it?

Reply:
I did not add it.